### DISCUSSION TITLE
Welcome to Contacts043 Community Discussions!

### DISCUSSION THREAD

**Discussion by AgrDeza** [AgrDeza/Contacts043_Community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Source336** [comment]
Welcome user to me for now!
![bandicam 2023-01-18 09-43-38-144](https://user-images.githubusercontent.com/115052285/213174684-89369c56-d35e-4b63-b167-170fcd0b0e54.jpg)

**Comment by Undoza6096** [comment]
![Screenshot_20230117-213309](https://user-images.githubusercontent.com/106401980/213263447-06225a57-ddfa-4829-abe8-3f34a9170aba.png)

**Reply by AgrDeza** [reply]
People?

**Comment by Undoza6096** [comment]
😜😜😜😜😜❤❤❤❤❤
❤❤❤❤❤😜😜😜😜😜

**Comment by Undoza6096** [comment]
> a

> B

> D

> H

> P

> Z

**Reply by Undoza6096** [reply]
![IMG-20230118-WA0117](https://user-images.githubusercontent.com/106401980/213473926-f5a4dc82-38a0-4be0-bb33-09538a040407.jpg)
Aocê
Bocê
Cocê
Gocê
Você
Vøç€

**Reply by TechCommunity-Adv** [reply]
can lol discuss by @Undoza6096
